code/datums/greyscale/config_types/greyscale_configs/greyscale_clothes.dm

```diff
@@ -423,6 +423,24 @@ SKYRAT EDIT END */
 	name = "Oversized Jacket (Worn)"
 	icon_file = 'icons/mob/clothing/suits/jacket.dmi'
 
+/datum/greyscale_config/jacket_lawyer
+	name = "Formal Suit Jacket"
+	icon_file = 'icons/obj/clothing/suits/jacket.dmi'
+	json_config = 'code/datums/greyscale/json_configs/jacket_lawyer.json'
+
+/datum/greyscale_config/jacket_lawyer/worn
+	name = "Formal Suit Jacket (Worn)"
+	icon_file = 'icons/mob/clothing/suits/jacket.dmi'
+
+/datum/greyscale_config/overalls
+	name = "Overalls"
+	icon_file = 'icons/obj/clothing/suits/utility.dmi'
+	json_config = 'code/datums/greyscale/json_configs/overalls.json'
+
+/datum/greyscale_config/overalls/worn
+	name = "Overalls (Worn)"
+	icon_file = 'icons/mob/clothing/suits/utility.dmi'
+
```

Comment:
### ❗**BEWARE**❗ 
This WILL conflict with the modular config/item of the same type. It was ported upstream entirely, now the base overalls are greyscale themselves. Recommend removing the modular overalls entirely and replacing spawns with this one.

Reply:
> @OrionTheFox you may want to add greyscale for the inhand icon too at some point. ![image](https://user-images.githubusercontent.com/13398309/261078822-c1d696a9-3cef-42a2-ba2a-45523c618af2.png)

o shit u rite